// express-resource/examples/controllers/forum.js
exports.index = function (req, res) {
    res.send([
        {
            "ownerID":7657865765,
            "accountKey":1000,
            "clientID":3579,
            "clientName":"ClientName",
            "journalTransactionID":40,
            "price":1000000,
            "quantity":2,
            "stationID":3333,
            "stationName":"JITA",
            "transactionDateTime":"2012.11.06 10:28",
            "transactionFor":"me",
            "transactionID":42342342423,
            "transactionType":"sell",
            "typeID":423423,
            "typeName":"Fenrir"},
        {
            "ownerID":223423423,
            "accountKey":1000,
            "clientID":1171119,
            "clientName":"Judas II",
            "journalTransactionID":20,
            "price":3321,
            "quantity":1,
            "stationID":9087,
            "stationName":"Nennamaila",
            "transactionDateTime":"2012.11.06 10:27",
            "transactionFor":"oo",
            "transactionID":756245,
            "transactionType":"buy",
            "typeID":224442,
            "typeName":"Manticore"}
    ]);
};

exports.new = function (req, res) {
    res.send('new forum');
};

exports.create = function (req, res) {
    res.send('create forum');
};

exports.show = function (req, res) {
    res.send('show forum ' + req.forum.title);
};

exports.edit = function (req, res) {
    res.send('edit forum ' + req.forum.title);
};

exports.update = function (req, res) {
    res.send('update forum ' + req.forum.title);
};

exports.destroy = function (req, res) {
    res.send('destroy forum ' + req.forum.title);
};

exports.load = function (id, fn) {
    process.nextTick(function () {
        fn(null, { title:'Ferrets' });
    });
};
